maquetado/index.html

```diff
@@ -11,37 +11,66 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
-  <div class="content">
-    <div class="content-center">
+  <div class="container">
+    <div class="wrap-technology">
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
+        <img class="logo" src="./assets/react_icon.svg" alt="tech logo">
+        <h4 class="technology-text">ReactJS</h4>
+        <h4 class="technology-text bold">Release:2013</h4>
         <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
-        <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+        <img class="logo" src="./assets/angular_icon.svg" alt="tech logo">
+        <h4 class="technology-text">AngularJS</h4>
```

Comment:
In most techs you have this set as `technology-title` and here is like `technology-text`. I think the first it's more accurate.